momentum_scanner_intraday.py

```diff
@@ -34,6 +34,14 @@ def token_technical_indicator(token):
 def token_price(token):
     return gecko.market_chart(token, days=1)["price"][-1]
 
+def token_volume_marketcap(token):
+    """
+    Days = 92 to make sure we are getting daily values
+    """
+    df = gecko.market_chart(token, days=92)
```

Comment:
92 pops the cache. Use `100`. Possibly we should add an assert in `gecko.py` that checks that `days in (1, 100)`.

Reply:
Changed to 100